### PR TITLE
add @ushicode profile

### PR DESCRIPTION
**Pull request by ushicode**
Added user .json file for new user @ushicode

### CODE REVIEW

File: Submissions/@ushicode.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Collins Ushi",
+    "img": "https://avatars.githubusercontent.com/u/4209868?v=4",
+    "email": "collinsushi@yahoo.com",
+    "links": {
+      "website": "https://github.com/ushicode",
+      "linkedin": "https://www.linkedin.com/in/collinsushi",
+      "github": "https://github.com/ushicode>"
```
Comment (ProsperousHeart) [inline Submissions/@ushicode.json:8]: please remove the >

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]

File: Submissions/@ushicode.json (hunk 1 above)
Comment (ushicode) [reply Submissions/@ushicode.json]: Changes made; the > has been removed from the code

**Comment by ushicode** [comment]
Changes effected

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
please remove the @ from the beginning of the filename

**Comment by ushicode** [comment]
File name modified; the symbol @ has been removed

**Review by ProsperousHeart — APPROVED** [review]

**Pull request merged**